src/antidote_crdt_set_rw.erl

```diff
@@ -144,7 +145,9 @@ unique() ->
 
 -spec update(downstream_op(), set()) -> {ok, set()}.
   update(DownstreamOp, RWSet) ->
-    {ok, apply_downstreams(DownstreamOp, RWSet)}.
+    RWSet1 = apply_downstreams(DownstreamOp, RWSet),
+    RWSet2 = [Entry || {_, {AddTokens, RemoveTokens}} = Entry <- RWSet1, AddTokens =/= [] orelse RemoveTokens =/= []],
+    {ok, RWSet2}.
```

Comment:
Hmm, if you remove these entries here, could you also simplify the `value` function? I think this is where this filtering is done at the moment...

Can you also update the documentation comment in line 30-40? That still describes the old version of the set. In the comment it should be described that we have an invariant here, that AddTokens and RemoveTokens can not be both empty.